### PR TITLE
Version 0.6.0

### PR DESCRIPTION
**Pull request by genotrance**

### CODE REVIEW

**Comment by dom96** [comment]
We need to fix that macOS crash first.

File: changelog.markdown [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Choosenim changelog
 
+## 0.6.0 - TBD
+
+The major new feature is default installation of 64-bit Nim
+binaries on Windows.
+
```
Comment (dom96) [inline changelog.markdown]:
```suggestion
Random selection of changes:
```

File: changelog.markdown [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Use git to `choosenim update devel` instead of deleting,
```
Comment (dom96) [inline changelog.markdown]:
```suggestion
* Git is now used for `choosenim update devel` instead of deleting,
```

File: changelog.markdown [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+  downloading and bootstrapping from scratch
+* Optionally add `~/.nimble/bin` to PATH on Windows on
+  `--firstInstall`
```
Comment (dom96) [inline changelog.markdown]:
```suggestion
  when using the `--firstInstall` flag.
```

File: changelog.markdown (hunk 1 above)
Comment (genotrance) [reply changelog.markdown]: This is actually all the changes :D

File: changelog.markdown (hunk 2 above)
Comment (genotrance) [reply changelog.markdown]: Done

File: changelog.markdown (hunk 3 above)
Comment (genotrance) [reply changelog.markdown]: Done

File: changelog.markdown (hunk 1 above)
Comment (dom96) [reply changelog.markdown]: So...
```suggestion
Changes:
```

Comment (genotrance) [reply changelog.markdown]: Done

**Pull request merged**